src/PhpSpreadsheet/Shared/CodePage.php

```diff
@@ -65,13 +67,29 @@ class CodePage
         65001 => 'UTF-8', //    Unicode (UTF-8)
     ];
 
+    /**
+     * @param string $codePage
+     *
+     * @return bool
+     */
+    public static function validate(string $codePage): bool
+    {
+        if (!in_array($codePage, self::$pageArray)) {
+            return false;
```

Comment:
Can be much simplified to a single line as `return (in_array($codePage, self::$pageArray, true));` with strict type checking for the codepage string as well

Reply:
fixed